src/main/java/com/github/tomakehurst/wiremock/client/WireMock.java

```diff
@@ -1047,4 +1030,31 @@ public GlobalSettings getGlobalSettings() {
   public static GlobalSettings getSettings() {
     return defaultInstance.get().getGlobalSettings();
   }
+
+  public enum JsonSchemaVersion {
```

Comment:
Should it be a `common.Json` nested class? It is a general-purpose class, and it looks a little odd here

Reply:
It's here because it's used in the DSL, and `WireMock` tries to be the entrypoint for that.

Perhaps this is something to consider more deeply in a 4.0 refactor rather than worrying too much now.